src/main/webapp/queue.js

```diff
@@ -44,6 +65,44 @@ function updateShareTab() {
       'lounge.html/?room_id='+ roomParameters +'" type="text" readonly</input>';
 }
 
+/* exported voteToSkip */
+// Tracks the user's choice to skip a video and alters firestore as needed
+function voteToSkip() {
+  voteBtnCount++;
+  if (voteBtnCount % 2 == 1) {
+    db.collection('rooms') // eslint-disable-line no-undef
+        .doc(roomParameters)
+        .collection('CurrentVideo')
+        .doc('PlaybackData')
+        .update({
+          skipVotes: firebase // eslint-disable-line no-undef
+              .firestore.FieldValue.increment(1),
+        }).then(function() {
+          console.log('votes incremented');
+        }).catch(function(error) {
+          console.log('voting caused an error: ', error);
+        });
+  } else if (voteBtnCount % 2 == 0) {
+    db.collection('rooms') // eslint-disable-line no-undef
+        .doc(roomParameters)
+        .collection('CurrentVideo')
+        .doc('PlaybackData')
+        .update({
+          skipVotes: firebase // eslint-disable-line no-undef
+              .firestore.FieldValue.increment(-1),
+        }).then(function() {
+          console.log('votes decremented');
+        }).catch(function(error) {
+          console.log('voting caused an error: ', error);
+        });
+  }
+  setTimeout(2000);
```

Comment:
why are we waiting for 2 sec? Can this be moved to a constant with relevant name like <X>_MILLI_SECONDS = 2000;

Reply:
I chose to wait for two seconds so users can't just spam the button, I can move it to a constant